package.json

```diff
@@ -126,7 +126,7 @@
     "@vue/test-utils": "^1.1.2",
     "babel-core": "^7.0.0-bridge.0",
     "css-loader": "4.3.0",
-    "electron": "^18.3.6",
+    "electron": "^20.3.3",
```

Comment:
Should we attempt bumping to `20.3.8`? It looks to be the latest version for Electron 20.x.

https://releases.electronjs.org/releases/stable?version=20